spec/features/directors/show_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'director show page', type: :feature do
+  # User Story 2, Parent Show (x2)
+  #
+  # As a visitor
+  # When I visit '/parents/:id'
+  # Then I see the parent with that id including the parent's attributes:
+  # - data from each column that is on the parent table
+
+  it 'displays the director name and attributes' do
+    director = Director.create!(name: "Darren Aronofsky", active: true, awards_won: 10)
```

Comment:
We could probably add another Director here too. To make sure it only displays the Director we expect. Obviously will pass but may as well.